src/index.js

```diff
@@ -1,7 +1,9 @@
 export {
   default as withTracking,
+  ReactTrackingContext,
   TrackingContextType,
 } from './withTrackingComponentDecorator';
 export { default as trackEvent } from './trackEventMethodDecorator';
 export { default as TrackingPropType } from './TrackingPropType';
+export { default as useTracking } from './useTracking';
```

Comment:
Out of curiosity, why enforce the `prefer default` eslint rule if defaults are then aliased later?

Reply:
It's a common style we use internally at NYT. In general we prefer default exports, but in this case we're re-exporting for external API.